charts/rocketmq-operator/templates/operator.yaml

```diff
@@ -27,7 +27,7 @@ spec:
       labels:
         name: rocketmq-operator
     spec:
-      serviceAccountName: rocketmq-operator
+      serviceAccountName: {{ template "rocketmq-operator.serviceAccountName" . }}
```

Comment:
This file is copied from that one in deploy directory(see the link below), pls also change the original file.
https://github.com/apache/rocketmq-operator/blob/a8665716c8028741ec106814798e39fe07e6dcaa/Makefile#L78C1-L88

Reply:
Sure, I'll go through it once I have time. Please bear with me for a couple of days.